.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "[python]": {
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-      "source.fixAll": true
+      "source.fixAll": "explicit"
```

Comment:
can we avoid comitting changes to this file for now?

Reply:
This was automatically changed by the newer version of VSCode FYI.